.dockerignore

```diff
@@ -0,0 +1,4 @@
+.git
```

Comment:
You might want to take a more general approach. Does ".*" can do the work?

Reply:
Done. ".*" does the job.
I check that inside the docker container we dont have any of those hidden files or folders under /code path